### DISCUSSION TITLE
GitHub rejects all our students even though we are verified by the Campus program, issue student emails and id

### DISCUSSION THREAD

**Discussion by SallyMcGrath** [community/community#68433] [GitHub Education]
### Hi everyone,

Dear GitHub

We have been enrolled in your GitHub Campus Program for some years now. Thank you so much for the work you do and share with us.  Unfortunately we've found it very hard to really use your offer, because you reject the vast majority of our students' applications for the developer pack. 

We issue a signed, dated enrollment letter on a letterheaded document. Our trainees are all members of our GitHub org, which is registered with GitHub Campus. We also created a verified student email system so they could use these resources, but for some reason they (nearly always) get rejected. Our trainees are largely refugees and asylum seekers. 83% live in poverty and 99% on a low income, so access to this pack would make a very large difference to their lives.  

I have searched the forum and found many questions, but few answers. Does the Developer Pack really exist? Is it only for people who don't need it?  Can you help us to understand how to get our students access to these resources?

**Comment by SallyMcGrath** [comment]
@samus-aran Is there any way to communicate with GitHub Campus directly any more? It feels quite awful to have to beg in public in this way and I'm concerned for the dignity of my trainees.

**Comment by samus-aran** [comment]
Hi @SallyMcGrath 

Thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value. Sometimes these changes do require schools to re-verify the school domains to ensure that we have the most up to date details so legitimate students can be verified without issue.

After reviewing the school's account we would recommend you (as faculty) to raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍  Include evidence of the domains that are used by the faculty and students so we can review and update the school details appropriately. This will aid in the process of getting the students verified more smoothly.

Hope this helps! Let us know how you get along 👍
